diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index d9240429f16424b36a2958fc3cb2fe0fedb7210b..fe165d528bf25a3a5f2e3e5506f7582d97fe8794 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -111,6 +111,7 @@ source_set("internal") {
       deps += [ "//chrome/browser/ui:overscroll_pref_manager" ]
     }
   }
+  deps += [ "//brave/browser/ui/browser_window/internal" ]
 }
 
 source_set("unit_tests") {
