diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 6176a350c6e70ab943c16f6acd481b3f28c882bb..907fe10124edab1e733792f3af9fbcae6c9380f2 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -456,6 +456,7 @@ void ToolbarView::Init() {
     media_button_ = container_view_->AddChildView(std::move(media_button));
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
