diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 9b0fea4c6ff8f43c695c437b5c295fcd41326d40..3692d5cc01265231bb993fcddb3a71c56e5e1f67 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3702,6 +3702,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
