diff --git a/content/browser/gpu/gpu_internals_ui.cc b/content/browser/gpu/gpu_internals_ui.cc
index dda46035cdcd25c54b4863de32ea0ed8204c18b6..841aed0ead6d14ee5da0a209e9cc40aef0e7eac0 100644
--- a/content/browser/gpu/gpu_internals_ui.cc
+++ b/content/browser/gpu/gpu_internals_ui.cc
@@ -816,6 +816,7 @@ base::Value::Dict GpuMessageHandler::GetClientInfo() {
            std::string("Skia/" STRINGIZE(SK_MILESTONE) " " SKIA_COMMIT_HASH));
   dict.Set("revision_identifier", GPU_LISTS_VERSION);
 
+  BRAVE_GPU_MESSAGE_HANDLER_ON_REQUEST_CLIENT_INFO
   return dict;
 }
 
